libs/coin-modules/coin-evm/src/cryptoAssetsStore.ts

```
@@ -0,0 +1,13 @@
+import {
+  CryptoAssetsStore,
+  CryptoAssetsStoreGetter,
+} from "@ledgerhq/types-live/lib/crypto-assets/type";
+
+let getStore: CryptoAssetsStoreGetter;
+export function setCryptoAssetsStoreGetter(cryptoAssetsStoreGetter: CryptoAssetsStoreGetter): void {
+  getStore = cryptoAssetsStoreGetter;
+}
+
+export function getCryptoAssetsStore(): CryptoAssetsStore {
```

We need to manage the "undefined getStore" case. We can throw an error as it is an unexpected state.

---

since it is a function, it will passed from the `live-common` package, which already handle this case 
https://github.com/LedgerHQ/ledger-live/blob/6c36be7a4c313c70ff3ff5712a294be85a974394/libs/ledger-live-common/src/bridge/crypto-assets/index.ts#L26